core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -236,4 +240,3 @@ registry.svc.ci.openshift.org/origin/4.7:vertical-pod-autoscaler quay.io/openshi
 registry.svc.ci.openshift.org/origin/4.7:vertical-pod-autoscaler-operator quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.7 quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.7.0 quay.io/openshift/origin-vertical-pod-autoscaler-operator:latest
 registry.svc.ci.openshift.org/origin/4.7:windows-machine-config-bootstrapper-tests quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:4.7 quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:4.7.0 quay.io/openshift/origin-windows-machine-config-bootstrapper-tests:latest
 registry.svc.ci.openshift.org/origin/4.7:windows-machine-config-operator-test quay.io/openshift/origin-windows-machine-config-operator-test:4.7 quay.io/openshift/origin-windows-machine-config-operator-test:4.7.0 quay.io/openshift/origin-windows-machine-config-operator-test:latest
-registry.svc.ci.openshift.org/ocp/4.7:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.7 quay.io/openshift/origin-operator-marketplace:4.7.0 quay.io/openshift/origin-operator-marketplace:latest
```

Comment:
@ecordell is there a reason this would disappear?  is this not built for 4.7?  It's still in the 4.8 list

Reply:
There is a 4.7 tag built 7 days ago

Reply:
This should still be present, there's no removal plan (yet).

Reply:
Can you regen

Reply:
This looks like a bug in the mapping generation, i'm investigating

Reply:
Sorry. This PR got closed by my wrong push.
I cannot undo it.

I will make another PR after your investigation.